### PR TITLE
fix: ensure kiva credit checkout errors bubble up and show a toast me…

### PR DESCRIPTION
**Pull request by mcstover**
…ssage

### CODE REVIEW

**Comment by mcstover** [comment]
With the expansion of `showCheckoutError` any previous error signatures should still be supported. I was able to verify this by seeing toast messages for `validatePreCheckout` errors returned which use the legacy signature. I did also check and saw that the Braintree Checkout errors are still working properly as they use their own error message handler.

**Comment by mcstover** [comment]
FYI @rubytidwell

File: src/plugins/checkout-utils-mixin.js [hunk 1]
```diff
@@ -114,36 +114,42 @@ export default {
...
 
 				/* eslint-disable max-len */
 				// update error messages for new checkout context (Original messages reference a different user flow)
-				if (error === 'ERROR_OWN_LOAN') {
+				if (errorType === 'ERROR_OWN_LOAN') {
 					// Original Message: As a Kiva borrower, you cannot support your own fundraising loan on Kiva.  Please go back to the basket to remove your loan.
 					errorMessage = 'As a Kiva borrower, you cannot support your own fundraising loan on Kiva. Please remove your loan before completing checkout.';
 				}
-				if (error === 'ERROR_OVER_DAILY_LIMIT') {
+				if (errorType === 'ERROR_OVER_DAILY_LIMIT') {
```
Comment (mcstover) [inline src/plugins/checkout-utils-mixin.js:138]: I was able to verify existing signature works with this error and that the new signature works with the UPC error.

**Review by emuvente — APPROVED** [review]
Nice! good find/fix

**Review by eddieferrer — APPROVED** [review]

**Pull request merged**